packages/core/src/scss/components/checkout/stripeLink/_stripeDefaultValues.scss

```diff
@@ -0,0 +1,13 @@
+@import "../../../../../src/app/ui/base";
+
+// We don't use these styles for the floating labels in the iframe.
+// These styles are in case of fallback for stripeIframe.scss
+// -----------------------------------------------------------------------------
+// stylelint-disable-next-line selector-pseudo-class-no-unknown
+:export {
+    border-radius: 4px;
```

Comment:
```suggestion
    border-radius: $global-radius;
```

Reply:
@bc-peng the idea is to avoid dynamic data, but let me know if it's causing any trouble we could remove the idea